blogposts/2021/optimizing-a-code-intelligence-commit-graph-1.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Optimizing a code intelligence commit graph (Part 1)"
+author: Eric Fritz
+authorUrl: https://eric-fritz.com
+description: "Read about our journey to enabling Sourcegraph to respond to requests on commits missing an index quickly, with precise results."
```

Comment:
@efritz I took a stab at a description (this appears on the card on /blog and when the post is shared on social). Feel free to edit if this doesn't capture it for you!

Reply:
```suggestion
description: "We enabled Sourcegraph to respond to requests for commits missing a code intelligence index quickly and with precise results. Read about our journey."
```

There were a lot of _to_ back to back so I reworded it a bit. How does this sound?